projects/ad713x_fmcz/.Xil/-20668-ADRIMBAR-L02/system.bda

```diff
@@ -0,0 +1,357 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
I believe this file was added by mistake 😄. Maybe we can add `.Xil` folder to gitignore.